app/models/run.rb

```diff
@@ -24,6 +24,7 @@ class Run < ApplicationRecord
   # dependent: :delete_all requires there be no child records that need to be deleted
   # If RunHistory is changed to have child records, change this back to just :destroy
   has_many :histories, class_name: 'RunHistory', dependent: :delete_all
+  has_one :highlight_suggestion
 
```

Comment:
Is there any merit into also associating this to a user directly?

Reply:
Yeah, it was just something that I was thinking of, not really a huge issue at this exact moment.